utils/build-script-impl

```diff
@@ -223,6 +223,12 @@ KNOWN_SETTINGS=(
     report-statistics                             "0"               "set to 1 to generate compilation statistics files for swift libraries"
     sil-verify-all                                "0"               "If enabled, run the SIL verifier after each transform when building Swift files during this build process"
     stdlib-deployment-targets                     ""                "space-separated list of targets to configure the Swift standard library to be compiled or cross-compiled for"
+    swift-sdks                                    ""                "space-separated list of SDKs to build, or empty to auto-detect which SDKs are available"
```

Comment:
Why did you add swift-sdks here. Are you still using this?

Reply:
Yes, I set it to avoid building for “OSX”.